packages/config/src/projects/optimism.ts

```diff
@@ -135,14 +130,7 @@ export const optimism: Project = {
               href: 'https://community.optimism.io/docs/users/gateway.html#withdrawing-back-to-l1',
             },
           ],
-          risks: [
-            // TODO: Uncomment after findings
-            // {
-            //   category: 'Funds can be lost if',
-            //   text: 'the centralized sequencer goes down and nobody can exit.',
-            //   isCritical: true,
-            // },
-          ],
+          risks: [EXITS.RISK_CENTRALIZED_BLOCK_PRODUCER],
```

Comment:
Would be great to have a reference here.

Reply:
Anyone can publish stateBatch as long as they have a bond. Currently BondManager that checks this is mocked (https://etherscan.io/address/0xCd76de5C57004d47d0216ec7dAbd3c72D8c49057#code) and vouch only for the OVM_Proposer EOA address

Reply:
Hmm we don't have this SC listed. I'll look into it